app/views/guidance/error_explanation.scala.html

```diff
@@ -0,0 +1,1601 @@
+            date is not valid. No further validation has been carried out.</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p>CDS12050</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p><strong>Procedure code is not valid</strong></p>
+            </td>
+        </tr>
+
+        <tr class="govuk-table__row" id="CDS12056">
+            <td class="govuk-table__cell">
+            <p>Relation error: the combination of elements is not allowed</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p>CDS12056</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p><strong>The combination of 2 values you entered is not allowed</strong></p>
+
+            <p>For example, a document code has an incompatible document status.</p>
+            </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+            <p>Relation error: The value supplied in Data Element 1 mandates the completion
+            of Data Element 2</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p>CDS12070</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p><strong>A required field is missing a value</strong></p>
+
+            <p>For example, you have entered a value that mandates another field to be
+            completed.</p>
+            </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+            <p>Relation error: Either Data Element 1 or Data Element 2 must be supplied,
+            but not both</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p>CDS12072</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p><strong>You declared 2 values that cannot be declared both together</strong></p>
+
+            <p>Check the <a href="https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide" target="_blank">CDS data element completion guide (opens in new tab)</a>.</p>
+            </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+            <p>Relation error: Data Element must be supplied at header or item level</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p>CDS12073</p>
+            </td>
+
+            <td class="govuk-table__cell">
+            <p><strong>The same data element has been completed twice</strong></p>
+
+            <p>Data elements <a href=
+            "https://www.google.com/url?q=https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide&amp;sa=D&amp;source=editors&amp;ust=1628512427759000&amp;usg=AOvVaw39QzzP9Wi9oYsf7iMWl8mM">may
```

Comment:
Are you sure this is the expected link? Clicking the link I land on a 'Redirect Notice' page by Google.

Reply:
Good catch - sorted.